### DISCUSSION TITLE
Reinersct RFID (basic) reader

### DISCUSSION THREAD

**Discussion by cncvikto** [MiczFlor/RPi-Jukebox-RFID#1374] [Q&A]
Dear Friends, Dear MiczFlor,
I have a ReinerSct basic rfid reader which is work with PC/SC principle. I see the Reader.py.pcsc in the scripts directory but It doesnt work for me. Does it work?

When I run:
`python ./RegisterDevice.py`
The error message is:
```
Traceback (most recent call last):
  File "./RegisterDevice.py", line 4, in <module>
    from Reader import get_devices
ImportError: cannot import name 'get_devices' from 'Reader' (/home/pi/RPi-Jukebox-RFID/scripts/Reader.py)
```

I see there is no **get_devices** function in Reader.py.pcsc.. (I rename this to Reader.py of course.) If I put get_devices function into that file, I can avoid error messages but I cant still choose rfidreader from the list, because the list is empty.
As I see the problem is that this rfid reader not present in /dev/input/. Only I see  here is the mice, or keyboard. 
What should I do dear friends?
Thank you in any advance!

**Comment by cncvikto** [comment]
Dear Friends, now I see clearly that I have to buy a nother type of rfid reader. But good news /experience for me that I learned a bit python yesterday! 
Phoniebox is a wonderful project, I want to make a phoniebox for blinds learning alphabets. 
Thank you MiczFlor, thank you Friends!

**Comment by cncvikto** [comment]
PS: I hope more blinds get vision with seeing with sound projekt (seeingwithsound.com) and after that a phoniebox for learning alphabets would be so usefull. Or they can use it for learning braille alphabets or identify other things!
Best wishes! 
Cncvikto

**Reply by MiczFlor** [reply]
Hi @cncvikto 
regarding the RFID reader: getting a different RFID reader will save you a lot of time. I think you made the right decision.
And thank you for the link to the seeing with sound project. That is very interesting. I remember a documentary many years ago which was working successfully on a similar (but different) project: they translated a video image into a matrix of pixels - which triggered sensations on the tongue. One person who was part of the experiment and who had lost his vision as a teenager, he actually described how these sensations at some point started to feel like a visionary input.
